lib/hooks/resolve/index.js

```diff
@@ -31,12 +32,11 @@ module.exports = settings => {
 
   return model => {
     return queue(model.data)
-      .then(response => pollChangelog(response.MessageId));
-    // .then(response => {
-    //   // insta-resolve
-    //   if (model.status === 'new') {
-    //     return pollChangelog(response.MessageId);
-    //   }
-    // });
+      .then(response => {
+        // insta-resolve
+        if (model.status === 'new') {
+          return pollChangelog(response.MessageId);
+        }
```

Comment:
I don't think this conditional should be there. Shouldn't we always wait for the message to resolve?

Reply:
Yeah you're probably right. I was only doing it for things that went straight from `new` to `done`, but thinking about it we probably also want to wait for the reply if a case is resolved by some other means

Reply:
_nods_